packages-exp/messaging-exp/src/interfaces/internal-dependencies.ts

```diff
@@ -27,3 +27,5 @@ export interface FirebaseInternalDependencies {
   installations: _FirebaseInstallationsInternal;
   analyticsProvider: Provider<FirebaseAnalyticsInternalName>;
 }
+
+export interface MessagingInternal {}
```

Comment:
Shouldn't it have the `getToken()` method?

Reply:
👍